run_distributed_examples.sh

```diff
@@ -54,9 +54,14 @@ function distributed_ddp() {
     uv run main.py || error "ddp example failed"
 }
 
+function distributed_minGPT() {
```

Comment:
I think it should be
```suggestion
function distributed_minGPT-ddp() {
```

you can revie the `run` function in `utils.sh`

Reply:
Done.